isis/src/base/apps/topds4/topds4.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="topds4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+  <brief>
+    Convert from ISIS3 cube to PDS 4 format using template XML files

```

Comment:
I think I'm advocating for changing the `ISIS3` to just `ISIS` since we're on ISIS 4.